README.md

```diff
@@ -77,3 +78,17 @@ conda env create -f cfret_main_env.yml
 ```bash
 conda activate cfret_data
 ```
+
+## Python and R analysis
+
+There are two different environments used for python (preprocessing steps) and 
+
+- [python_analysis_env](./python_analysis_env.yml): This environment is for use in python specific notebooks, like the preprocessing and analyze data modules.
+- [R_analysis_env](./R_analysis_env.yml): This environment is for use in R specific notebooks, specifically for visualizing data.
```

Comment:
What about the `cfret_main_env.yml`? Should that be included here?

Reply:
I have updated this to be better at describing the different environment, great catch!